tmt-app/src/Components/BobaBot.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import BobaBotLarge from "../Images/art/boba_bot_Large.svg";
+import { Box, Typography } from "@mui/material";
+
+function BobaBot({ text }) {
+  return (
+    <Box
+      sx={{
+        alignItems: "center",
+        display: "inline-flex",
+        position: "relative",
+      }}
+    >
+      <img src={BobaBotLarge} alt={"Mascot Icon"} />
+      <Box
+        sx={{
+          alignItems: "center",
+          display: "flex",
+          height: "40%",
+          left: "15%",
+          position: "absolute",
+          top: "10%",
+          width: "70%",
```

Comment:
Does this look okay on different view ports?

Reply:
I'm also concerned about the use of unclamped, static percentages here